templates/registration/profile.html

```diff
@@ -3,17 +3,27 @@
 
 {% block content %}
     <h1>Profile</h1>
+    <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_username %}!</h2>
     {% if artists %}
+        <div class="investments">
```

Comment:
I think you have the wrong number of `<div>` tags. Personally I think it would be easier to see what's going on if all of your open tags were indented exactly as far as your close tags.